apps/mobile/src/store/accounts/recurse-accounts.ts

```diff
@@ -0,0 +1,79 @@
+// from extension src/app/common/account-restoration/account-restore.ts
```

Comment:
this shouldn't be added directly to mobile, instead though of as a distinct module that I think should like in `/crypto` pkg, the refactored out to extension

Reply:
I will do this once I can get it to work properly